cmd/root.go

```diff
@@ -129,6 +134,7 @@ func Execute(analyticsTracker analytics.Tracker, version string) {
 		FlagsClient:        flags.NewClient(version),
 		MembersClient:      members.NewClient(version),
 		ProjectsClient:     projects.NewClient(version),
+		GenericClient:      &http.Client{Timeout: time.Second * 3},
```

Comment:
i guess this isn't required yet but wanted to get feedback on where we'd be passing it in